src/Umbraco.Web.BackOffice/Controllers/LanguageController.cs

```diff
@@ -25,15 +20,11 @@ public class LanguageController : UmbracoAuthorizedJsonController
     {
         private readonly ILocalizationService _localizationService;
         private readonly IUmbracoMapper _umbracoMapper;
-        private readonly GlobalSettings _globalSettings;
 
-        public LanguageController(ILocalizationService localizationService,
-            IUmbracoMapper umbracoMapper,
-            IOptionsSnapshot<GlobalSettings> globalSettings)
+        public LanguageController(ILocalizationService localizationService, IUmbracoMapper umbracoMapper)
         {
             _localizationService = localizationService ?? throw new ArgumentNullException(nameof(localizationService));
             _umbracoMapper = umbracoMapper ?? throw new ArgumentNullException(nameof(umbracoMapper));
-            _globalSettings = globalSettings.Value ?? throw new ArgumentNullException(nameof(globalSettings));
```

Comment:
Same here

Reply:
I've also updated the `Language` constructor, so all previous public constructors are now marked as obsolete 👍🏻